File: Diplom/Diplom/Models/NotificationModels/NotificationInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Diplom.Models.NotificationModels
{
    public class NotificationInfo
    {
        public int NotificationId { get; set; }
        public int UserId { get; set; }
        public int SourceId { get; set; }
        public NotificationType NotificationType { get; set; }
        public string Message { get; set; }
        public DateTime CreateDate { get; set; }
        public bool IsChecked { get; set; }
    }
}

File: Diplom/Diplom/ClientApp/src/app/models/notifications/notificationInfo.ts
import {NotificationType} from "./notification";

export class NotificationInfo {

  constructor(notificationId: number, userId: number, sourceId: number, notificationType: NotificationType,
              message: string, isChecked: boolean, createDate: Date, routeString: string) {
    this.notificationId = notificationId;
    this.userId = userId;
    this.sourceId = sourceId;
    this.notificationType = notificationType;
    this.message = message;
    this.isChecked = isChecked;
    this.createDate = createDate;
    this.routeString = routeString;
  }

  notificationId: number;
  userId: number;
  sourceId: number;
  notificationType: NotificationType;
  message: string;
  isChecked: boolean;
  createDate: Date;
  routeString: string;
}
